Makefile

```diff
@@ -1,85 +1,41 @@
-DESCRIPTION = CS50 Library for Java
-MAINTAINER = CS50 <sysadmins@cs50.harvard.edu>
-OLD_NAME = lib50-java
-NAME = libcs50-java
-VERSION = 2.0.0
-
-BUILD_DIR = build
-DEB_DIR = $(BUILD_DIR)/deb
-DOCS_DIR = docs
-JAR_DIR = $(BUILD_DIR)/usr/java/packages/lib/ext
-PACMAN_DIR = $(BUILD_DIR)/pacman
-RPM_DIR = $(BUILD_DIR)/rpm
-
-.PHONY: all
-all: clean deb pacman rpm
-
-.PHONY: bash
-bash:
-	docker run -i --rm -t -v "$(PWD):/root" cs50/cli
+DESTDIR ?= /usr/local/share/java
+VERSION = 2.0.1
 
 .PHONY: build
 build: clean Makefile src/edu/harvard/CS50.java
-	mkdir -p "$(JAR_DIR)"
-	javac -d "$(BUILD_DIR)" src/edu/harvard/CS50.java
-	jar cvf "$(JAR_DIR)/cs50.jar" -C "$(BUILD_DIR)" .
-	find "$(BUILD_DIR)" -type d -exec chmod 0755 {} +
-	find "$(BUILD_DIR)" -type f -exec chmod 0644 {} +
+	mkdir -p build/usr/share/java
+	javac -d build src/edu/harvard/CS50.java
+	cp src/edu/harvard/CS50.java build/edu/harvard
+	jar cvf build/usr/share/java/cs50.jar -C build .
```

Comment:
> Should use DESTDIR here, no? 

`DESTDIR` is used in [`make install`](https://github.com/cs50/libcs50-java/blob/93a4c1f2aedbb11008f3821e603037fbf0437845/Makefile#L36).

> what's build/usr/share/java/ used for exactly? Does it end up in the JAR?

`/usr/share/java` ends up in the deb (not the JAR). It's recommended by [lintian](https://lintian.debian.org/) that Java libraries end up in `/usr/share/java`, so this just preps the structure for deb packaging. If you use `make install` with `DESTDIR`, JAR should end up in `DESTDIR` instead.

Unfortunately though, Java doesn't look for JARs under `/usr/share/java` by default, so users would probably have to specify classpath somehow. Earlier we were installing the JAR under `/usr/java/packages/lib/ext`, which is searched for JARs by default, but lintian complains about it.

https://lintian.debian.org/tags/jar-not-in-usr-share.html